Formula/math-comp.rb

```diff
@@ -0,0 +1,46 @@
+class MathComp < Formula
+  desc "Mathematical Components for the Coq proof assistant"
+  homepage "https://math-comp.github.io/math-comp/"
+  url "https://github.com/math-comp/math-comp/archive/mathcomp-1.6.4.tar.gz"
+  sha256 "c672a4237f708b5f03f1feed9de37f98ef5c331819047e1f71b5762dcd92b262"
+  head "https://github.com/math-comp/math-comp.git"
+
+  option "without-docs", "Don't generate or install documentation"
+
+  depends_on "ocaml"
+  depends_on "coq"
+
+  def install
+    ENV["COQBIN"] = "#{Formula["coq"].bin}/"
+    (buildpath/"mathcomp/Makefile.coq.local").write(<<~EOF)
+      COQLIB=#{lib}/coq/
+    EOF
+
+    cd "mathcomp" do
+      system "make", "MAKEFLAGS=#{ENV["MAKEFLAGS"]}"
+      system "make", "install"
+
+      (elisp/"ssreflect").install "ssreflect/pg-ssr.el"
```

Comment:
Are you sure there's no install target for this?

Also, this seems weird. The path will be `share/emacs/site-lisp/math-comp/ssreflect`. Is `share/emacs/site-lisp/math-comp` not good enough?

Reply:
No make target for elisp.

Reply:
Thoughts on the second question?

Reply:
Changed the install path.